[BUG] kbcli basic cmd runtime panic

**Describe the bug**
kbcli basic cmd runtime panic
```
➜  kubeblocks git:(main) bin/kbcli
panic: runtime error: index out of range [1] with length 1

goroutine 1 [running]:
github.com/apecloud/kubeblocks/internal/cli/cmd.NewCliCmd.func2(0x14000c81c88?, {0x107fc8ab0, 0x0, 0x0})
        /Users/alal/kubeblocks/internal/cli/cmd/cli.go:162 +0x114
k8s.io/component-base/cli.run.func3(0x14000971b00?, {0x107fc8ab0, 0x0, 0x0})
        /Users/alal/kubeblocks/vendor/k8s.io/component-base/cli/run.go:137 +0x68
github.com/spf13/cobra.(*Command).execute(0x14000971b00, {0x1400006e240, 0x0, 0x0})
        /Users/alal/kubeblocks/vendor/github.com/spf13/cobra/command.go:915 +0x4fc
github.com/spf13/cobra.(*Command).ExecuteC(0x14000971b00)
        /Users/alal/kubeblocks/vendor/github.com/spf13/cobra/command.go:1068 +0x35c
github.com/spf13/cobra.(*Command).Execute(...)
        /Users/alal/kubeblocks/vendor/github.com/spf13/cobra/command.go:992
k8s.io/component-base/cli.run(0x14000971b00)
        /Users/alal/kubeblocks/vendor/k8s.io/component-base/cli/run.go:146 +0x2a4
k8s.io/component-base/cli.RunNoErrOutput(...)
        /Users/alal/kubeblocks/vendor/k8s.io/component-base/cli/run.go:84
main.main()
        /Users/alal/kubeblocks/cmd/cli/main.go:31 +0x20

```
**To Reproduce**
`kbcli` in bash

**Expected behavior**
A clear and concise description of what you expected to happen.

**Screenshots**
If applicable, add screenshots to help explain your problem.
![image](https://github.com/apecloud/kubeblocks/assets/101848970/11be0524-486e-47f2-9bd2-059db187ee18)

--- a/internal/cli/cmd/cli.go
+++ b/internal/cli/cmd/cli.go
@@ -159,6 +159,9 @@ A Command Line Interface for KubeBlocks`,
 
 			commandPath := cmd.CommandPath()
 			parts := strings.Split(commandPath, " ")
+			if len(parts) < 2 {
+				return nil
+			}
 			subCommand := parts[1]
 			if cloudCmds[subCommand] && !auth.IsLoggedIn() {
 				return fmt.Errorf("use 'kbcli login' to login first")

